### DISCUSSION TITLE
Thanks and Drag & drop

### DISCUSSION THREAD

**Discussion by 2ant** [rds1983/Myra#358] [Q&A] [answered]
Hi, thank you for this awesome lib. 

I'm trying to do some drag & drop and i don't know  how to.

How can i make a button draggable ? 

Ultimately, i wolud like to be able to create something like this :  https://imgur.com/jR9adEq

that i saw here : https://github.com/rds1983/Myra/pull/225

**Comment by rds1983** [comment, marked as answer]
Code for making a button draggable:
```c#
  button.DragHandle = button;
  button.DragDirection = DragDirection.Both;  
```

**Reply by 2ant** [reply]
Thank you very much for your fast answer !

How can i detect when a drag start and when the widget is dropped ?

If you can help me with this, i think i will have all i need to do a Diablo like inventory where i can drag and drop stuff.

Thanks again for Myra, i'm so glad i can discover C#, monogame and that i found a nice UI library.

**Reply by rds1983** [reply]
For now, use events TouchDown/TouchUp to detect when the drag starts/ends. Eventually I'll add dedicated events.
And you're welcome!
